pages/new-collective-page.js

```diff
@@ -29,17 +28,6 @@ const IncognitoUserCollective = dynamic(
   { loading: Loading },
 );
 
-/** Add global style to enable smooth scroll on the page */
-const GlobalStyles = createGlobalStyle`
-  html {
-    scroll-behavior: smooth;
-  }
-
-  section {
-    margin: 0;
```

Comment:
Where is that supported now?

Reply:
i have left the margin for section in global style